website/source/docs/drivers/docker.html.md

```diff
@@ -201,6 +201,26 @@ The `docker` driver supports the following configuration in the job spec:
       ]
     }
     ```
+* `volume_driver` - (Optional) The name of the volume driver used to mount
+  volumes. Must be used along with `volumes`.
+  Using a `volume_driver` also allows to use `volumes` with a named volume as
+  well as regular paths.
+
+  ```hcl
+    config {
+      volumes = [
+        # Use absolute paths to mount arbitrary paths on the host
+        "/path/on/host:/path/in/container",
+
+        # Use relative paths to rebind paths already in the allocation dir
+        "relative/to/task:/also/in/container",
```

Comment:
Remove these other examples that are already included above as I think the `name-of-the-volume` example is the important one when showing how to use `volume_driver`